contracts/consumer/virtual-staking/src/contract.rs

```diff
@@ -69,7 +71,7 @@ impl VirtualStakingContract<'_> {
     }
 
     /// The caller of the instantiation will be the converter contract
-    #[msg(instantiate)]
+    #[sv::msg(instantiate)]
     pub fn instantiate(&self, ctx: InstantiateCtx) -> Result<Response, ContractError> {
```

Comment:
Since the contract is now using some custom message and query you have to use them in the methods signatures. Otherwise you get this errors with conversion from `DepsMut<'_, Empty>` to `DepsMut<'_, VirtualStakeCustomQuery>`

```suggestion
    pub fn instantiate(&self, ctx: InstantiateCtx<VirtualStakeCustomQuery>) -> Result<Response<VirtualStakeCustomMsg>, ContractError> {
```